core/src/test/resources/util/test-application-multi.yml

```diff
@@ -0,0 +1,29 @@
+spring:
+  profiles:
+    active: dev,qa
+
+---
+
+spring:
+  profiles: dev
```

Comment:
yes, something like this as shown in https://github.com/spring-projects/spring-boot/blob/master/spring-boot-samples/spring-boot-sample-profile/src/main/resources/application.yml 

Also, if it could not have multiple values, why did you split it then in the code above ? Also I believe it could be even a YAML list (same like for activeProfiles)